src/main/java/org/sagebionetworks/bridge/sdk/models/PagedResourceList.java

```diff
@@ -1,45 +1,82 @@
 package org.sagebionetworks.bridge.sdk.models;
 
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
 import java.util.List;
+import java.util.Map;
 
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
 
 public class PagedResourceList<T> {
+    private static final String OFFSET_KEY_FILTER = "offsetKey";
     private final List<T> items;
-    private final int offsetBy;
+    private final Integer offsetBy;
     private final int pageSize;
     private final int total;
-    private final String emailFilter;
+    private final Map<String,String> filters = Maps.newHashMap();
 
     @JsonCreator
-    public PagedResourceList(@JsonProperty("items") List<T> items, @JsonProperty("offsetBy") int offsetBy,
-            @JsonProperty("limitTo") int pageSize, @JsonProperty("total") int total,
-            @JsonProperty("emailFilter") String emailFilter) {
+    PagedResourceList(
+            @JsonProperty("items") List<T> items, 
+            @JsonProperty("offsetBy") Integer offsetBy,
+            @JsonProperty("pageSize") int pageSize, 
+            @JsonProperty("total") int total) {
         this.items = items;
         this.offsetBy = offsetBy;
         this.pageSize = pageSize;
         this.total = total;
-        this.emailFilter = emailFilter;
+    }
+
+    /**
+     * A convenience method for adding filters without having to construct an intermediate map of filters.
+     * e.g. PagedResourceList<T> page = new PagedResourceList<T>(....).withFilterValue("a","b");
+     */
+    PagedResourceList<T> withFilter(String key, String value) {
+        if (isNotBlank(key) && isNotBlank(value)) {
+            filters.put(key, value);
+        }
+        return this;
+    }
+    /**
+     * Convenience method for adding the DDB key as a filter. The key must be returned to retrieve 
+     * the next page of DDB records.
+     */
+    PagedResourceList<T> withOffsetKey(String offsetKey) {
+        return withFilter(OFFSET_KEY_FILTER, offsetKey);
     }
```

Comment:
Removing these, SDK users never make paging objects. Added offsetKey to verify it is deserialized correctly (as a kind of filter value).